src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/annotatedregion/SimpleAnnotatedGenomicRegionCollection.java

```diff
@@ -0,0 +1,261 @@
+                collection.getContigColumnName(), collection.getStartColumnName(), collection.getEndColumnName());
+    }
+
+    /**
+     * Create a collection from components.
+     *
+     * @param regions regions to use in the resulting collection.  Never {@code null}.
+     * @param samFileHeader SAMFileHeader to include in the collection.  Represents the sample(s)/references that were used for these segments.
+     *                      {@code null} is allowed.
+     * @param annotations List of annotations to preserve in the regions.  Never {@code null}.  These are the annotations that will be written.
+     * @param contigColumnName contig column name to use if the collection is written.  Can't be {@code null}, empty, or a number.
+     * @param startColumnName start column name to use if the collection is written.  Can't be {@code null}, empty, or a number.
+     * @param endColumnName end column name to use if the collection is written.  Can't be {@code null}, empty, or a number.
+     * @return collection based on the inputs
+     */
+    public static SimpleAnnotatedGenomicRegionCollection create(final List<SimpleAnnotatedGenomicRegion> regions,
+                                                                                    final SAMFileHeader samFileHeader,
+                                                                                    final List<String> annotations,
+                                                                                    final String contigColumnName,
+                                                                                    final String startColumnName,
+                                                                                    final String endColumnName) {
+
+        Utils.nonNull(regions);
+        Utils.nonNull(annotations);
+        XsvLocatableTableCodec.validateLocatableColumnName(contigColumnName);
+        XsvLocatableTableCodec.validateLocatableColumnName(startColumnName);
+        XsvLocatableTableCodec.validateLocatableColumnName(endColumnName);
+
+        return new SimpleAnnotatedGenomicRegionCollection(samFileHeader, annotations, Collections.emptyList(), regions,
+                contigColumnName, startColumnName, endColumnName);
+    }
+
+
+
+    /** Create a collection based on the contents of an input file and a given config file.  The config file must be the same as
+     * is ingested by {@link XsvLocatableTableCodec}.
+     *
+     * @param input readable path to use for the xsv file.  Must be readable.  Never {@code null}.
+     * @param inputConfigFile config file for specifying the format of the xsv file.  Must be readable.  Never {@code null}.
+     * @param headersOfInterest Only preserve these headers.  These must be present in the input file.  This parameter should not include the locatable columns
```

Comment:
Why `headersOfInterest` here and `columnsOfInterest` elsewhere?

Reply:
Done